ldap/servers/slapd/upgrade.c

```diff
@@ -98,32 +98,6 @@ upgrade_AES_reverpwd_plugin(void)
     return uresult;
 }
 
-#ifdef RUST_ENABLE
```

Comment:
Could this be a combination ifdef RUST_ENABLE && ENTRYUUID_AUTOMATIC or similar, so that we can keep this automatic for distros that want it?

Reply:
Sure no problem...

Reply:
One issue here is that the plugin still needs to be off by default.  Because on upgrade the server is restarted (%post) before the new subpackage is installed.  So the plugin library will not exist and the server will fail to start during the upgrade.